src/Sarif.Viewer.VisualStudio/ErrorList/SarifTableControlEventProcessorProvider.cs

```diff
@@ -93,26 +89,25 @@ public override void PreprocessSelectionChanged(TableSelectionChangedEventArgs e
             public override void PreprocessNavigate(ITableEntryHandle entryHandle, TableEntryNavigateEventArgs e)
             {
                 ThreadHelper.ThrowIfNotOnUIThread();
-                SarifErrorListItem sarifResult;
 
-                if (!TryGetSarifResult(entryHandle, out sarifResult))
+                if (!TryGetSarifResult(entryHandle, out SarifErrorListItem sarifErrorListItem))
                 {
                     SarifViewerPackage.SarifToolWindow.Control.DataContext = null;
                     return;
                 }
 
                 e.Handled = true;
 
-                if (sarifResult.HasDetails)
+                if (sarifErrorListItem.HasDetails)
                 {
                     SarifViewerPackage.SarifToolWindow.Show();
                 }
 
                 // Navigate to the source file of the first location for the defect.
-                if (sarifResult.Locations?.Count > 0)
+                if (sarifErrorListItem.Locations?.Count > 0)
                 {
-                    sarifResult.Locations[0].NavigateTo(false);
-                    sarifResult.Locations[0].ApplyDefaultSourceFileHighlighting();
+                    sarifErrorListItem.Locations[0].NavigateTo(false);
```

Comment:
Indexing twice for the first element is hoping that the optimize will make it go away.  I don't think that does.  Then again FirstOrDefault() like above